test/Quake/circuit-cutting.qke

```diff
@@ -0,0 +1,41 @@
+// ========================================================================== //
+// Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                 //
+// All rights reserved.                                                       //
+//                                                                            //
+// This source code and the accompanying materials are made available under   //
+// the terms of the Apache License 2.0 which accompanies this distribution.   //
+// ========================================================================== //
+
+// RUN: cudaq-opt --help | grep cut-quake &> /dev/null && cudaq-opt %s --cut-quake=k=3 &> %basename.txt && FileCheck --input-file=%basename.txt %s || echo 0
```

Comment:
Right now this test is skipped, since the build and test environment doesn't have the necessary graph partition library. I assume you intend to use this one: https://github.com/KarypisLab/METIS? I have some concerns with relying on this library (e.g. there is no CI on their repo), and I'd rather not take in code that is not used and tested anywhere. Let's chat offline about options here.